tools/realsense-viewer/CMakeLists.txt

```diff
@@ -62,6 +68,17 @@ if(DEFINED OpenCV_DIR  AND  IS_DIRECTORY ${OpenCV_DIR})
     set(DEPENDENCIES ${deps} ${OpenCV_LIBS})
     include_directories( ../../wrappers/opencv )
 
+    list(APPEND DELAYED
+        opencv_highgui341.dll
```

Comment:
Hard-coding names here creates coupling with specific openCv version (3.4). It will not affect users linking with other versions, so the delayed loading will be skipped.

Reply:
That is true. That said, SDK installer comes bundled with **specific** version of OpenCV and OpenVINO and we want to protect installer users first. I couldn't think of a way to make this generic. 
With all that said, delay loading is not the most important part of this PR. If you don't do delay loading, you still get a message, "cannot find realsense2.dll". You get less info, and there is no "Report" button, but otherwise its sufficient.